AutoGrade/reportviews.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from django.contrib.admin.views.decorators import staff_member_required
+from django.utils import timezone
+
+from .models import *
+from .reports import *
```

Comment:
Don't use `*` here (put the name of class and function here), it get complicated most of the time and other TAs also have to work on this.

Reply:
Done